edge-agent/docker/linux/amd64/Dockerfile

```diff
@@ -1,7 +1,7 @@
-ARG base_tag=2.1.17-alpine3.11
+ARG base_tag=3.1.4-alpine3.11
 FROM azureiotedge/azureiotedge-runtime-base:1.2-linux-amd64 as builder
 
-FROM mcr.microsoft.com/dotnet/core/runtime:${base_tag}
```

Comment:
We changed to use aspnet as it references from framework package rather than NuGet package since asp.net 3.1, is it right?

Reply:
Yes

Reply:
FYI, the size of dotnet/core/aspnet (347MB) is 20M more than dotnet/core/runtime (327MB) on Windows.  We may need to mention this size change in release notes.

Reply:
Please check about docker image size differences for other platforms.  thanks.

Reply:
Even though the aspnet base image adds ~20 MB, the total image sizes are down somewhat.

1.0.9 Hub: 154MB
1.0.9 Agent: 141MB
PR Hub: 138MB
PR Agent: 138MB